sgv2-docsapi/src/main/java/io/stargate/sgv2/docsapi/service/write/db/DeleteSubDocumentArrayQueryBuilder.java

```diff
@@ -28,7 +28,34 @@
 import io.stargate.sgv2.docsapi.service.util.DocsApiUtils;
 import java.util.List;
 
-/** Deletes any array elements in the given document sub-path. */
+/**
+ * Deletes any array elements in the given document sub-path.
+ *
+ * <p>For example, given the document:
+ *
+ * <pre>
+ * { "a": 1, "b": [2, 3] }
+ * </pre>
+ *
+ * That gets stored as the following rows:
+ *
+ * <pre>
+ * key | p0 | p1    | p2 | dbl_value
+ * --- +----+-------+----+-----------
+ * xyz |  a |       |    |         1
+ * xyz |  b | [000] |    |         2
```

Comment:
I am probably wrong here, but I thought index was expressed with 6 digits earlier?

Reply:
This is now configurable via `documentProperties.maxArrayLength()` (see `addBindValues` below). But yes the default is indeed 6.
I went with 3 to keep the example compact, but actually it doesn't matter, I'll change it.

Reply:
Ok.

Configurability both makes sense and seems dangerous: mostly because changing the setting could make existing data invalid, if I understand it correctly, if changing the setting when restarting. Unless there is something in code to convert data back and forth.

Reply:
Configuration is needed as anyway we have different props atm for the serverless and classic for example..